docs/data/material/discover-more/roadmap/roadmap.md

```diff
@@ -42,41 +42,42 @@ Here are the top priorities:
 
 Here are the components we will work on being supported in the MUI ecosystem:
 
+- ✅ Released as stable
 - 🧪 Close to becoming stable, already released as unstable
 - 🛠 Work in progress, will be or already released as unstable
 - ⏳ Planning to build
 
-| Name                       | Product  | Status |
-| :------------------------- | :------- | :----- |
-| Advanced Layout            | MUI X    | ⏳     |
-| Carousel                   | MUI X    | ⏳     |
-| Charts                     | MUI X    | ⏳     |
-| Data Grid                  | MUI X    | 🧪     |
-| Date Picker                | MUI X    | 🧪     |
-| Date Range Picker          | MUI X    | 🧪     |
-| Dropdown                   | MUI Core | ⏳     |
-| Dropzone                   | MUI X    | ⏳     |
-| File Upload                | MUI X    | ⏳     |
-| Gantt Chart                | MUI X    | ⏳     |
-| Gauge                      | MUI X    | ⏳     |
-| Image                      | MUI Core | ⏳     |
-| Masonry                    | MUI Core | 🧪     |
-| Navbar                     | MUI Core | ⏳     |
-| Nested Menu                | MUI X    | ⏳     |
-| NProgress                  | MUI Core | ⏳     |
-| Numeric Input              | MUI Core | ⏳     |
-| Rich Text Editor           | MUI X    | ⏳     |
-| Scheduler                  | MUI X    | ⏳     |
-| Scrollspy                  | MUI Core | ⏳     |
-| Sparkline                  | MUI X    | ⏳     |
-| Timeline                   | MUI Core | 🧪     |
-| Tree select                | MUI X    | ⏳     |
-| Tree View                  | MUI X    | 🧪     |
-| Tree View - Checkbox       | MUI X    | ⏳     |
-| Tree View - Drag & Drop    | MUI X    | ⏳     |
-| Tree View - Multiselect    | MUI X    | 🧪     |
-| Tree View - Virtualization | MUI X    | ⏳     |
-| Window Splitter            | MUI X    | ⏳     |
+| Name                                                                     | Product  | Status |
+| :----------------------------------------------------------------------- | :------- | :----- |
+| Advanced Layout                                                          | MUI X    | ⏳     |
+| Carousel                                                                 | MUI X    | ⏳     |
+| Charts                                                                   | MUI X    | ⏳     |
+| [Data Grid](/x/react-data-grid/)                                         | MUI X    | ✅     |
+| [Date Picker](/x/react-date-pickers/date-picker/)                        | MUI X    | ✅     |
+| [Date Range Picker](/x/react-date-pickers/date-range-picker/)            | MUI X    | ✅     |
```

Comment:
We might want to replace this with just "Date and Time Pickers", or link to more components from here instead.
cc @flaviendelangle

Reply:
I don't know
On one hand we want to make very clear that we support advanced pickers, not just the usual `DatePicker` and `TimePicker`.
So not saying "range" at all on this page is a regression in clarity

But on the other hand, the current one totally lack the "time" part which is also important...

Reply:
cc @LukasTy @richbustos @joserodolfofreitas

Reply:
On one hand, it would make sense listing each picker components for better clarity:
- Date Picker ✅ 
- Time Picker ✅ 
- Date Time Picker ✅ 
- Date Range Picker ✅ 
- Time Range Picker ⏳
- Date Time Range Picker ⏳

This would allow for better visibility as to what is done and what do we still plan to do, but on the other hand, it does create a lot of noise... 🙈 
Some other libraries do not even differentiate `Range` pickers in their docs, it's often just a different picker mode, thus
- Date & Time Pickers ✅ 
- Time & Date and Time Range Pickers ⏳ 
could also work, albeit, with worse visibility 🙈 

TL.DR. IMHO, we are not endorsing our components enough, hence, listing them here one by one does make sense, especially given the doc structure and components we expose. 🤔

Reply:
I like the idea of listing more picker components. How about fields?

Reply:
As much as I do agree that fields are a fundamental part of pickers, I'd say that in this context, it would probably make sense to omit them. 🤔